### PR TITLE
Migrations, deliveryman created.

### PR DESCRIPTION
**Pull request by rynkwn**

### CODE REVIEW

File: app/models/deliveryman.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Deliveryman < ActiveRecord::Base
+  
+  validates :name, presence: true
+  validates :phone, presence: true
+  validates :bank_info, presence: true
```
Comment (BenDrews) [inline app/models/deliveryman.rb]: We should also validate the format of phone numbers and bank information. Not something that needs to be done right now though.

File: db/migrate/20150616183237_create_receipts.rb [hunk 2]
```diff
@@ -9,6 +9,10 @@ def change
       t.text :item_type
       t.string :status # Order Sent -> Paid For -> In Transit -> Delivered -> Complete
       t.text :checkin_code # Allows non-logged in users to check status of their receipt
+      t.text :seller_availability # Available times seller can drop off item
+      t.text :buyer_availability # Available times buyer can drop off item.
```
Comment (BenDrews) [inline db/migrate/20150616183237_create_receipts.rb]: Is it necessary to have both seller and buyer availability?

Comment (rynkwn) [inline db/migrate/20150616183237_create_receipts.rb]: Yes. They're separate entities, both of which need to be available during their respective phase of the delivery process in order for the delivery to be successful.

Comment (BenDrews) [inline db/migrate/20150616183237_create_receipts.rb]: Oh, I misinterpreted the columns as belonging to the user model. This definitely makes sense.

File: app/models/deliveryman.rb (hunk 1 above)
Comment (kevinchen93) [inline app/models/deliveryman.rb]: See #99

File: db/migrate/20150616183237_create_receipts.rb [hunk 3]
```diff
@@ -9,6 +9,10 @@ def change
...
+      t.integer :claimed # 0 if Unclaimed. 1 if Claimed by a deliveryperson.
```
Comment (kevinchen93) [inline db/migrate/20150616183237_create_receipts.rb]: Since it's binary, why not:
```ruby
t.boolean :claimed
```

**Pull request merged**